cloud/cloud.go

```diff
@@ -187,8 +207,16 @@ func GetManager(ctx context.Context, env evergreen.Environment, mgrOpts ManagerO
 // GetManagerOptions gets the manager options from the provider settings object for a given
 // provider name.
 func GetManagerOptions(d distro.Distro) (ManagerOpts, error) {
+	if len(d.ProviderSettingsList) > 1 {
+		return ManagerOpts{}, errors.Errorf("distro should be modified to have only one provider settings")
+	}
 	if IsEc2Provider(d.Provider) {
-		return getEC2ManagerOptions(d)
+		s := &EC2ProviderSettings{}
+		if err := s.FromDistroSettings(d, ""); err != nil {
+			return ManagerOpts{}, errors.Wrapf(err, "error getting EC2 provider settings from distro")
```

Comment:
isn't this not necessarily ec2

Reply:
It is because it's in a block `if IsEC2Provider(d.Provider)`